pcap_bw.py

```diff
@@ -52,11 +58,11 @@ def inet_to_str(inet):
         hostbw = { i: { j: 0 for j in HOSTS } for i in HOSTS }
         for ts, hdr, buf in pcap:
             if prevts == 0:
-                s = ", ".join(
+                s = ",".join(
```

Comment:
The spaces were deliberate -- makes it easier to inspect the CSV by eye; is there a particular reason for removing them?

Reply:
When reading the csv into pandas dataframes you end up with odd column titles (like ' 10.0.0.1:10.0.0.1')

Reply:
pandas is python right? in which case, `' a string with prefix and trailing spaces '.strip()` is your friend :)